docroot/sites/admissions.uiowa.edu/modules/admissions_core/sass/student-profile.scss

```diff
@@ -1,12 +1,54 @@
+@import "../../../../../themes/custom/uids_base/uids/assets/scss/_variables.scss";
+@import "../../../../../themes/custom/uids_base/uids/assets/scss/_utilities.scss";
+@import "../../../../../themes/custom/uids_base/uids/assets/scss/_grid.scss";
+
 .block-views-blockstudent-card-block-student-grid {
+  &[class*="grid"].hide-details {
+    .view-content.list-container {
+      gap: 0 !important;
+    }
+
+    .card__body {
+      display: none;
+    }
+
+    &.grid__mobile {
+      &.grid--threecol--33-34-33,
+      &.grid--twocol--50-50 {
+        .list-container {
+          @include display-grid;
+
+          .card {
+            margin-bottom: 0;
+          }
+        }
+      }
+
+      &.grid--threecol--33-34-33 {
```

Comment:
I can't get my local to work to test this, but are these grid includes needed if they are already being defined in https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/components/grid.scss?

Reply:
if i recall, we couldn't use that because of the `@include grid-base` affecting the styles when we were trying to maintain grid for mobile